packages/ses/src/permits.js

```diff
@@ -1600,3 +1572,39 @@ export const permitted = {
 
   '%InitialGetStackString%': fn,
 };
+
+try {
+  // eslint-disable-next-line
+  new FERAL_FUNCTION('async function* (){}') // A string, so Metro doesn't throw creating release bundle
+  assign(permitted, {
+    '%InertAsyncGeneratorFunction%': {
+      // Properties of the AsyncGeneratorFunction Constructor
+      '[[Proto]]': '%InertFunction%',
+      prototype: '%AsyncGenerator%',
+    },
+
+    '%AsyncGenerator%': {
```

Comment:
this is causing `yarn test` to fail

```console
  TypeError: Expected permit object at whitelist.%AsyncGenerator%
    at completePrototypes (packages/ses/src/intrinsics.js:99:15)
    at repairIntrinsics (packages/ses/src/lockdown.js:280:3)
    at globalThis.lockdown (packages/ses/src/lockdown-shim.js:16:28)
    at lockdown (packages/lockdown/pre.js:86:5)
    at packages/lockdown/commit-debug.js:14:1
```

Reply:
fixed in https://github.com/endojs/endo/pull/2334/commits/74a580c1acd6010c79323cfcce539bfac109a032